### PR TITLE
Update CFP page for DC28

### PR DESCRIPTION
**Pull request by BenGardiner**
Here's a proposed change to the CFP page text and an update to the CFP link.

### CODE REVIEW

File: CFP.md [hunk 1]
```diff
@@ -7,15 +7,15 @@ layout: page
 ### Call for papers for DEF CON 28 opening soon!
 We want to give you a platform to talk on!  Have a cool hack you want to show off?  Want to teach something awesome?  Feel like blabbering on in front of a crowd?  Let us know!
 
-We're most interested in the following:
+This year more than others we  are seeking anything and everything related to making your own use that is hands-on. Do you have a training or workshop that includes a hands-on component? Perhaps there is a specific meetup of a special (hardware) interest group you would like to lead? Topics that we are always interested in include:
```
Comment (kbembedded) [inline CFP.md]: Looks like a double space between "we" and "are." I don't think this shows up on the final webpage, but if so, it should be fixed.

Comment (BenGardiner) [reply CFP.md]: good eye

**Comment by kbembedded** [comment]
Looks good to me!

**Review by ShortTie — APPROVED** [review]
Looks great! Thanks again for keeping on top of this!

**Comment by BenGardiner** [comment]
thank you both. I'll push the merge button here when I send out the mailers.

**Pull request merged**